packages/app/components/settings/tabs/phone.tsx

```diff
@@ -1,21 +1,17 @@
-import { useMemo, useCallback } from "react";
-import { Platform, ScrollView } from "react-native";
+import { useMemo } from "react";
+import { Platform } from "react-native";
 
 import { TabScrollView } from "@showtime-xyz/universal.collapsible-tab-view";
 import { useRouter } from "@showtime-xyz/universal.router";
 import { View } from "@showtime-xyz/universal.view";
 
-import {
-  SettingPhoneNumberSlotHeader,
-  SettingsPhoneNumberSkeletonSlot,
-  SettingsPhoneNumberSlot,
-  SettingsPhoneNumberSlotPlaceholder,
-} from "app/components/settings/settings-phone-number-slot";
-import { SlotSeparator } from "app/components/settings/slot-separator";
+import { EmptyPlaceholder } from "app/components/empty-placeholder";
+import { SettingsPhoneNumberItem } from "app/components/settings/settings-phone-number-item";
```

Comment:
rename a lot of `Slot` to other name because my OCD, if use Slot named component make me feel uncomfortable because `vue.js` 🤣